src/framework/global/internal/interactive.cpp

```diff
@@ -237,7 +237,9 @@ io::paths_t Interactive::selectMultipleDirectories(const QString& title, const i
 
 QColor Interactive::selectColor(const QColor& color, const QString& title)
 {
+    shortcutsRegister.get()->setActive(false);
     QColor selectedColor = QColorDialog::getColor(color, nullptr, title);
+    shortcutsRegister.get()->setActive(true);
```

Comment:
would be nicer to guarantee this would happen even if getColor throws an exception, but in general the codebase doesn't really make use of that sort of pattern. Is it likely we'd continue using the QColorDialog in the long term, vs QML-based dialogs?